### ISSUE TITLE
What do embedded binaries do and where are they used ?

### ISSUE THREAD

**Issue by makingglitches** [VUE/VUE#63]
Kind of curious what these files under the jnilib subdirectory do.

![Screenshot from 2024-01-07 13-36-27](https://github.com/VUE/VUE/assets/53788764/7cfd14fd-8e93-4c61-8a67-210f0c1f1ac0)

**Comment by rpavlik** [comment]
those are custom swing look and feels, can be omitted iirc. They were pretty easy for me to learn about with a web search, which I did as a part of my process of replacing ant with gradle and updating all the deps that I could.

**Comment by makingglitches** [comment]
@rpavlik  have you used the current source version ? I am noting there were 36 commits after the last working version, presently I'm looking at this damn thing and getting a bug where after i draw anything on the map all the menu options get grayed out... are you aware of this rather familiar bug ?

**Comment by rpavlik** [comment]
Yes, I have a branch in my repo based on the latest source that builds and seems to work fine.

**Comment by sfraize** [comment]
libquaqua provided native file dialogs on MacOS (instead of the java ones) -- it was REALLY fantastic for the years it worked (I don't think it can still be used)-- I imagine there must be a replacement for this out there somewhere, but who knows.  VUE would attempt to find the native libraries on startup, and use them if it could.

**Comment by makingglitches** [comment]
@sfraize  I have noticed some of the apple specific code is not building.
Please have a look at my fork if you would.
